flask_api/app.py

```diff
@@ -116,3 +123,7 @@ def create_url_adapter(self, request):
                 self.config['SERVER_NAME'],
                 script_name=self.config['APPLICATION_ROOT'] or '/',
                 url_scheme=self.config['PREFERRED_URL_SCHEME'])
+
+    @staticmethod
+    def get_minor_version():
```

Comment:
I wonder if this function might make more sense to live in `compat.py` and be reworked to something like `is_flask_0_10`?